sources/packages/web/src/components/aest/students/modals/DeclineApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <modal-dialog-base
+    :showDialog="showDialog"
+    title="Are you sure you want to decline the change?"
+  >
+    <template #content>
+      <p>
+        The request for change in your application from your institution will
+        not move forward. Are you sure you want to decline it?
+      </p>
+    </template>
+    <template #footer>
+      <footer-buttons
+        primaryLabel="Decline now"
+        @secondaryClick="cancel"
+        @primaryClick="declineChange"
+      />
+    </template>
+  </modal-dialog-base>
+</template>
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { ref, defineComponent } from "vue";
+import { useModalDialog } from "@/composables";
+import { UpdateApplicationOfferingChangeRequestAPIInDTO } from "@/services/http/dto";
+import { ApplicationOfferingChangeRequestStatus } from "@/types";
+
+export default defineComponent({
+  components: {
+    ModalDialogBase,
+  },
+  setup() {
+    const { showDialog, showModal, resolvePromise } = useModalDialog<
+      UpdateApplicationOfferingChangeRequestAPIInDTO | boolean
+    >();
+    const declineApplicationOfferingChangeRequestModal = ref(
+      {} as UpdateApplicationOfferingChangeRequestAPIInDTO,
+    );
+    const cancel = () => {
+      resolvePromise(false);
+    };
+    const declineChange = async () => {
+      declineApplicationOfferingChangeRequestModal.value.applicationOfferingChangeRequestStatus =
+        ApplicationOfferingChangeRequestStatus.DeclinedByStudent;
```

Comment:
do we need this check?

Reply:
@ann-aot https://github.com/bcgov/SIMS/pull/2196#discussion_r1302311282

Reply:
same qn as , https://github.com/bcgov/SIMS/pull/2196#discussion_r1309364531